README.md

```diff
@@ -1 +1,57 @@
-# proffessional-portfolio
\ No newline at end of file
+# proffessional-portfolio
+
+This is a Concert site for events. The site is fully responsive.
+
+#loom link
+![]loom:(https://www.loom.com/share/6fd2e393d5394e59b75ffdbdda2e1a2c)
+
+# Original idea
+Original design idea by Cindy Shin. Big shoutout to her.
```

Comment:
- [ ] Kindly place this recognition under the **acknowledgement section** of your ReadMe. This will keep the ReadMe more organized